src/Bundle/Renderer/TwigGridRenderer.php

```diff
@@ -113,14 +113,14 @@ public function renderAction(GridViewInterface $gridView, Action $action, $data
     public function renderFilter(GridViewInterface $gridView, Filter $filter)
     {
         $template = $this->getFilterTemplate($filter);
+        $filterName = str_replace('\\', '', $filter->getName());
```

Comment:
![image](https://github.com/user-attachments/assets/48acf2e1-8ba9-40ba-8d7c-c17fc2bb488b)

Reply:
It seems a bit dirty sadly